src/services/messages.js

```diff
@@ -14,11 +14,20 @@ export async function sendMessage(email, status = 0) {
   return parseData(res);
 }
 
-export function subscribe(onMessage = (_message) => {}) {
+export function subscribe(handleMessageReceived = (_message) => {}) {
+  const mySubscription = client
+    .from('messages')
+    .on('INSERT', (payload) => {
+      handleMessageReceived(payload.new);
+      console.log('Change Received', payload.new);
+    })
+    .subscribe();
+  return parseData(mySubscription);
   // TODO: Subscribe to changes for the `messages` table
```

Comment:
Here we can solely call .subscribe() and remove our return statement on line 25.